build.gradle

```diff
@@ -30,11 +30,14 @@ version = '1.0'
 
 repositories {
     mavenCentral()
+    maven { url "https://jitpack.io" }
 }
 
 dependencies {
     implementation 'com.github.ajalt.clikt:clikt:3.3.0'
-    implementation "com.github.ajalt.mordant:mordant:2.0.0-beta3"
+    implementation 'com.github.ajalt.mordant:mordant:2.0.0-beta3'
```

Comment:
Was hat sich in der Zeile geändert? :D

Reply:
Die Anführungszeichen :P

Reply:
Ah, ty^^ Dachte schon iwie line-endings oder iwie sowas :D